packages/fx-core/src/common/deps-checker/internal/ngrokChecker.ts

```diff
@@ -41,14 +40,13 @@ export class NgrokChecker implements DepsChecker {
   }
 
   public getDepsInfo(): Promise<DepsInfo> {
-    const details = new Map<string, string>();
-    details.set(binFolderKey, this.getNgrokBinFolder());
     return Promise.resolve({
       name: ngrokName,
       isLinuxSupported: true,
       installVersion: installPackageVersion,
+      binFolders: [this.getNgrokBinFolder()],
```

Comment:
Curious why ngrok bin folders contain only one path, but func core tool bin folders contain two path?

Reply:
You can learn more from `funcToolChecker.getPortableFuncBinFolders()`

Reply:
I know func bin folders handle the npm6 issue. My point is that if this issue happens for ngrok?

Reply:
My understanding is - func is pointed to the npm top-level bin folder which handled by npm, but ngrok is pointed to the internal binary folder which handled by ngrok post-install script (Lucky that node_modules folder does not have compatible issue).